extension/src/test/suite/experiments/util.ts

```diff
@@ -129,6 +130,35 @@ export const buildExperiments = ({
   }
 }
 
+export const buildExperimentsWebview = async (inputs: {
+  availableNbCommits?: { [branch: string]: number }
+  disposer: Disposer
+  dvcRoot?: string
+  expShow?: ExpShowOutput
+  gitLog?: string
+  remoteExpRefs?: string
+  rowOrder?: { branch: string; sha: string }[]
+  stageList?: string | null
+}) => {
+  const all = buildExperiments(inputs)
+  const { experiments, messageSpy } = all
+  await experiments.isReady()
+  const webview = await experiments.showWebview()
+  messageSpy.restore()
+  const instanceMessageSpy: typeof messageSpy = spy(webview, 'show')
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ;(experiments as any).webviewMessages.sendWebviewMessage()
```

Comment:
[F] Ideally we will move towards this instance spy pattern and be able to remove the original spy altogether. This will however take a bit of work so we'll need to move there incrementally.

Reply:
spying on the instance is more reliable than spying on `BaseWebview.prototype`